internal/pkg/mapper/defaultbucket/local_load.go

```diff
@@ -0,0 +1,59 @@
+package defaultbucket
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/keboola/keboola-as-code/internal/pkg/filesystem"
+	"github.com/keboola/keboola-as-code/internal/pkg/model"
+	"github.com/keboola/keboola-as-code/internal/pkg/utils/orderedmap"
+)
+
+// MapAfterLocalLoad - replace placeholders with default buckets in IM.
+func (m *defaultBucketMapper) MapAfterLocalLoad(recipe *model.LocalLoadRecipe) error {
+	config, ok := recipe.Object.(*model.Config)
+	if !ok {
+		return nil
+	}
+
+	err := m.visitStorageInputTables(config, m.replacePlaceholderWithDefaultBucket)
+	if err != nil {
+		m.Logger.Warnf(`Warning: %s`, err)
+	}
+	return nil
+}
+
+func (m *defaultBucketMapper) replacePlaceholderWithDefaultBucket(config *model.Config, inputTableSource string, inputTable *orderedmap.OrderedMap) error {
+	if !strings.HasPrefix(inputTableSource, "{{:default-bucket:") {
+		return nil
+	}
+
+	sourceWithoutPrefix := strings.TrimPrefix(inputTableSource, "{{:default-bucket:")
```

Comment:
Tu je jeden problem, ze ak `TrimPrefix` prefix nenajde, tak sa nic nedeje ...
tj. prejde ti aj `abcdef}}.xyz`.

Pridal by som podmienku `if strings.HasPrefix`, alebo by som to vyriesil  s `regexp`.

Reply:
Tu podmínku mám o tři řádky víš. 😄

Ale to jsem se chtěl zeptat, co se týče obecně takovýhodle vytahování ze stringu, tak pokud to je takhle celkem jednoduchý, tak jsem to v phpku raději dělal obdobně jako teď stringovýma funkcema který jsou v php nativně v céčku a jsou rychlejší než matchování regexem (nebo to tak aspoň bývávalo, už vlastně nevím jestli jsem si z toho neudělal spíš cargo cult 😃), tak jestli je to tak i v Go, nebo si o tom myslíš.

Reply:
:man_facepalming: